xds/src/main/java/io/grpc/xds/orca/OrcaServiceImpl.java

```diff
@@ -23,39 +23,47 @@
 import com.github.xds.service.orca.v3.OrcaLoadReportRequest;
 import com.google.common.annotations.VisibleForTesting;
 import com.google.protobuf.util.Durations;
+import io.grpc.BindableService;
+import io.grpc.ServerServiceDefinition;
 import io.grpc.SynchronizationContext;
 import io.grpc.stub.ServerCallStreamObserver;
 import io.grpc.stub.StreamObserver;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-final class OrcaServiceImpl extends OpenRcaServiceGrpc.OpenRcaServiceImplBase {
+final class OrcaServiceImpl implements BindableService {
```

Comment:
Why not extend OpenRcaServiceGrpc.OpenRcaServiceImplBase?

Reply:
To hide the protos, allowing service proto to change in the future?

Reply:
Well, right now this isn't a public class. I think I had mentioned making it public, and yeah, then BindableService makes sense.

Reply:
could you remind me why we want to make it public?